platform/src/pages/analytics/_components/OverView.jsx

```
@@ -36,23 +37,101 @@ const OverView = () => {
+    // If no sites are selected, reset data
+    if (chartData.chartSites.length === 0) {
+      setLineData([]);
+      setBarData([]);
+      setLoading(false);
+      return;
+    }
+
+    const { startDate, endDate } = dateRange;
+
+    // Validate that startDate and endDate are valid Date objects
+    if (!(startDate instanceof Date) || isNaN(startDate)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)